language_detector/main.py

```diff
@@ -2,7 +2,41 @@
 
 """This is the entry point of the program."""
 
+'''
+
+def detect_language(text, languages):
+    """Returns the detected language of given text."""
+    spanish = 0
+    german = 0
+    english = 0
+    txt_lst = text.split()
+    for word in txt_lst:
+        if word in languages[0]['common_words']:
+            spanish += 1
+        elif word in languages[1]['common_words']:
+            german += 1
+        elif word in languages[2]['common_words']:
+            english += 1
+
+    if spanish > (german and english):
+        return "Spanish"
+    elif german > (spanish and english):
+        return "German"
+    else:
+        return "English"
+'''
+
+
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    from collections import defaultdict
```

Comment:
Should be doing your imports at the top of your file as per [PEP8](http://legacy.python.org/dev/peps/pep-0008/#imports) (Python style guildelines):